src/pages/philippines/regions/index.tsx

```diff
@@ -35,7 +36,8 @@ const PhilippinesRegions: React.FC = () => {
         .join(' ')
```

Comment:
Great solution!

However, It might be better to solve the root cause which is fixing regions names in the JSON files that are in upper case.